content/blog/2020-01-ggplot2-3-3-0/index.Rmarkdown

```diff
@@ -0,0 +1,231 @@
+---
+title: "ggplot2 3.3.0"
+author: "Thomas Lin Pedersen"
+date: '2020-01-07'
+description: |
+  The next iteration of ggplot2 has just been released on CRAN, packed with new 
+  features and bug fixes. Read all about what's new here.
+photo:
+  author: Steve Harvey
+  url: https://unsplash.com/photos/xWiXi6wRLGo
+slug: ggplot2-3-3-0
+categories: ["package"]
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE, 
+  comment = "#>"
+)
+```
+
+We're so happy to announce the release of ggplot2 3.3.0 on CRAN. ggplot2 is a system for declaratively creating graphics, based on *The Grammar of Graphics*. You provide the data, tell ggplot2 how to map variables to aesthetics, what graphical primitives to use, and it takes care of the details.
+
+This release is a substantial release with both big new features and internal rewrites. It contains most of the work Dewey Dunnington did as part of his internship at RStudio, along with contributions from many others. Dewey will continue working on ggplot2 as part of the core team, and we are very lucky to have him.
```

Comment:
```suggestion
This release is substantial, with both big new features and internal rewrites. It contains most of the work [Dewey Dunnington](https://github.com/paleolimbot) did as part of his internship at RStudio, along with contributions from many others. Dewey will continue working on ggplot2 as part of the core team, and we are very lucky to have him.
```

Reply:
@paleolimbot what do you think about linking to your post on the ed. site (https://education.rstudio.com/blog/2019/10/a-summer-of-rstudio-and-ggplot2/) above? (We could kill the link to your GH profile, just to avoid having too much click-i-ness in that case; I wanted to add some ref. to you for the curious)